modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/delegate/SourceModelLoaderDelegate.java

```diff
@@ -6,21 +6,36 @@
  */
 package org.mule.runtime.module.extension.internal.loader.delegate;
 
+import static java.lang.String.format;
+import static java.util.Collections.emptyList;
+import static java.util.Optional.empty;
+import static java.util.Optional.of;
+import static java.util.stream.Collectors.toList;
 import static org.mule.runtime.extension.api.property.BackPressureStrategyModelProperty.getDefault;
 import static org.mule.runtime.module.extension.internal.loader.ModelLoaderDelegateUtils.requiresConfig;
 import static org.mule.runtime.module.extension.internal.loader.parser.java.notification.NotificationModelParserUtils.declareEmittedNotifications;
+import static org.mule.runtime.module.extension.internal.loader.utils.JavaModelLoaderUtils.declareMetadataResolverFactoryModelProperty;
```

Comment:
Same as before, either this is in the wrong utils class, or should not be used here.